synapse/federation/sender/__init__.py

```diff
@@ -415,12 +416,10 @@ def __init__(self, hs: "HomeServer"):
             / hs.config.ratelimiting.federation_rr_transactions_per_room_per_second
         )
 
-        # wake up destinations that have outstanding PDUs to be caught up
-        self._catchup_after_startup_timer: Optional[
-            IDelayedCall
-        ] = self.clock.call_later(
-            CATCH_UP_STARTUP_DELAY_SEC,
+        # Regularly wake up destinations that have outstanding PDUs to be caught up
+        self.clock.looping_call(
```

Comment:
Do we have to worry about multiple `_wake_destinations_needing_catchup()` calls stacking up if they don't finish within the hour?

Does `looping_call` handle that de-duplication for us? (I don't see anything in the docstring)